packages/terra-paginator/src/ControlledPaginator.jsx

```diff
@@ -202,6 +202,7 @@ class Paginator extends React.Component {
           onClick={this.handlePageChange(previousPageIndex)}
           onKeyDown={this.handleOnKeyDown(previousPageIndex)}
         >
+          <span className={cx('visually-hidden')}>Previous</span>
```

Comment:
Action-header has the translations for Previous and Next. Can we add the internationalization for these? https://github.com/cerner/terra-core/blob/master/packages/terra-action-header/translations/en-GB.json#L4

Reply:
Translations will be added in a follow-up PR to resolve #1493